pkg/constants/constants.go

```diff
@@ -56,8 +56,8 @@ const (
 	GovcPasswordKey    = "GOVC_PASSWORD"
 	SnowCredentialsKey = "AWS_B64ENCODED_CREDENTIALS"
 	SnowCertsKey       = "AWS_B64ENCODED_CA_BUNDLES"
-	NutanixUsernameKey = "NUTANIX_USER"
-	NutanixPasswordKey = "NUTANIX_PASSWORD"
+	NutanixUsernameKey = "EKSA_NUTANIX_USERNAME"
+	NutanixPasswordKey = "EKSA_NUTANIX_PASSWORD"
```

Comment:
Since we are changing the env var, you will have to set the env vars that CAPX expects here https://github.com/aws/eks-anywhere/blob/main/pkg/providers/nutanix/env.go#L17-L22 similar to [this](https://github.com/aws/eks-anywhere/blob/main/pkg/providers/vsphere/envars.go#L13-L27)

Reply:
will be added thru separate PR